pages/dkp/kommander/2.3/cli/dkp/create/bootstrap/index.md

```diff
@@ -26,7 +26,7 @@ dkp create bootstrap [flags]
   -h, --help                             help for bootstrap
       --http-proxy string                HTTP proxy for CAPI controllers
       --https-proxy string               HTTPS proxy for CAPI controllers
-      --kind-cluster-image string        Kind node image for the bootstrap cluster (default "mesosphere/konvoy-bootstrap:v0.0.0-dev.0")
+      --kind-cluster-image string        Kind node image for the bootstrap cluster (default "mesosphere/konvoy-bootstrap:v2.3.2-rc.2")
```

Comment:
There are a few of these in here, where -rc.2 shows - these need removed, don't they?

Reply:
@tillt @msdolbey - See question above - don't we need to remove these -rc.2 items before we merge this?

Reply:
Yes - you are correct @Colin1841 - these will need to be retagged as the correct version of konvoy/dkp/this bootstrap, etc.

Reply:
we should re run the automation, not update manually. @tillt

Reply:
Removed all rc.2 references.